### PR TITLE
feat(concepts): backups

### PR DESCRIPTION
**Pull request by Sijoma**
## Description

"Hot" Backups documentation. 

@felix-mueller , it would be great if you can contribute to this. Especially on the purpose of the backups. E.g. "it is not for data archival" - I unfortunately have not found the section were we discussed this already. 

@ColRad, is there something for the Console API where we should link to? 

@hisImminence, I could also use your special wording magic to push this one through and make it good for the release.

## When should this change go live?

- [x] This change is not yet live and should not be merged until `13th of June` 
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

File: docs/components/concepts/backups.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+id: backups
+title: "Backups"
+description: "Learn more about Backups in Camunda Platform 8 SaaS."
+---
+
+You can use the backup feature of Camunda Platform 8 SaaS to regularly back up the state of all of its components (Zeebe, Operate, Tasklist, and Optimize) with _zero downtime_. In case of failures that lead to data loss, you can request to restore the backup.
+
+A backup of Camunda Platform 8 SaaS consists of a backup of Zeebe, Operate, Tasklist, Optimize, and the backup of exported Zeebe records in Elasticsearch. Since the data of these applications are dependent on each other, it is important that the backup is consistent across all components. Therefore, the backup of a Camunda Platform 8 Cluster is taken as a whole.
+
+With backups, you can capture snapshots of your data and applications while they are actively in use, resulting in zero downtime or disruption to your operations. Backups are designed specifically for disaster recovery purposes.
```
Comment (felix-mueller) [inline docs/components/concepts/backups.md:11]: Backups are designed specifically for disaster recovery purposes. Backups should not be used for archival of process data.

Comment (felix-mueller) [reply docs/components/concepts/backups.md:11]: @Sijoma maybe like this is already sufficient for now

Comment (Sijoma) [reply docs/components/concepts/backups.md]: Thanks for adding that. I think thats the clarity I needed for that. 👍  

Also I'm happy for more review on the content. A LGTM too :D.

**Review by felix-mueller — DISMISSED** [review]
lgtm

File: docs/components/concepts/backups.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+:::caution
+Exercise caution when deleting clusters to avoid unintended loss of backups.
+:::
+
+Your cluster generation needs to be greater or equal to 8.2.4-alpha1 to support backups.
+
+## Rate Limits
+
+The backup feature is available exclusively to [<span class="badge badge--enterprise-only">Camunda Platform Enterprise</span>] users. Each cluster has a limit of three backups. To ensure system stability backup operations are subject to rate limits. Specifically, you can perform a backup operation every 5 hours. However, it is possible to delete a backup and create a new one before the 5-hour rate limit expires.
```
Comment (akeller) [inline docs/components/concepts/backups.md:25]: Is the rate limiting feature for Enterprise customers only? Or hot backups?

Comment (akeller) [reply docs/components/concepts/backups.md]: In my `review fixes` commit I moved the badge to the top of the page and removed this sentence.

File: docs/components/concepts/backups.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+id: backups
+title: "Backups"
+description: "Learn more about Backups in Camunda Platform 8 SaaS."
+---
+
+You can use the backup feature of Camunda Platform 8 SaaS to regularly back up the state of all of its components (Zeebe, Operate, Tasklist, and Optimize) with _zero downtime_. In case of failures that lead to data loss, you can request to restore the backup.
+
+A backup of Camunda Platform 8 SaaS consists of a backup of Zeebe, Operate, Tasklist, Optimize, and the backup of exported Zeebe records in Elasticsearch. Since the data of these applications are dependent on each other, it is important that the backup is consistent across all components. Therefore, the backup of a Camunda Platform 8 Cluster is taken as a whole.
+
+With backups, you can capture snapshots of your data and applications while they are actively in use, resulting in zero downtime or disruption to your operations. Backups are designed specifically for disaster recovery purposes. Backups should not be used for archival of process data.
```
[the remark this answers is not in the document]
Comment (Sijoma) [reply docs/components/concepts/backups.md:13]: Good point. I guess we will see and I will then adjust if we see that customers don't get their purpose? I want to trust our customers first.

File: docs/components/concepts/backups.md (hunk 2 above)
Comment (Sijoma) [reply docs/components/concepts/backups.md]: Ohh good point. Do you have a recommendation where it should go? Right at the beginning after the explanation what they are?

Backups are enterprise only.

File: docs/components/concepts/backups.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+---
+id: backups
+title: "Backups"
+description: "Learn more about Backups in Camunda Platform 8 SaaS."
+---
+
+You can use the backup feature of Camunda Platform 8 SaaS to regularly back up the state of all of its components (Zeebe, Operate, Tasklist, and Optimize) with _zero downtime_. In case of failures that lead to data loss, you can request to restore the backup.
+
+A backup of Camunda Platform 8 SaaS consists of a backup of Zeebe, Operate, Tasklist, Optimize, and the backup of exported Zeebe records in Elasticsearch. Since the data of these applications are dependent on each other, it is important that the backup is consistent across all components. Therefore, the backup of a Camunda Platform 8 Cluster is taken as a whole.
```
[the remark this answers is not in the document]
Comment (Sijoma) [reply docs/components/concepts/backups.md]: I like that too - more clear this way. 🙇

File: docs/components/concepts/backups.md (hunk 2 above)
Comment (akeller) [reply docs/components/concepts/backups.md]: I moved it to the top of the page 😄

**Review by akeller — APPROVED** [review]

**Pull request merged**